tests/integration/test_multiple_rule_providing_apps.py

```diff
@@ -73,24 +74,22 @@ async def test_first_relation_one_alert_rule(ops_test):
         await ops_test.model.wait_for_idle(
             apps=[app_name, rules_app], status="active", timeout=1000
         )
-
+    global rules_after_relation
```

Comment:
Could this global be avoided, while maintaining clarity, by merging two test functions into one?

Reply:
As the test steps are written, it is not possible to do so without losing clarity.